apps/condo/domains/acquiring/schema/MultiPayment.test.js

```diff
@@ -360,7 +360,9 @@ describe('MultiPayment', () => {
             })
             test('Cannot accept payments with different acquiring', async () => {
                 const { billingIntegration, admin, payments, client } = await makePayerAndPayments()
-                const [integration] = await createTestAcquiringIntegration(admin, [billingIntegration])
+                const [integration] = await createTestAcquiringIntegration(admin)
+                const newBillingIntegrationGroup = 'test'
```

Comment:
Why this needed here? Seems like it's not affected the test results...
1. Payments are created with integration `A`
2. Multipayment **must** be created with integration `A`

Reply:
If you want to test group logic create a separate test for that